recipes/python-logstash-async/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "python-logstash-async" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ad68a05aac6c7d2d065b3dd8ba2e2b9bda59d103c75db2aa11b815e03a67c151
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pylogbeat
+    - requests
+    - python >=3.6
+
+test:
+  imports:
+    - logstash_async
```

Since upstream `__init__.py` doesn't import much, it would be good to add something like 
```suggestion
    - logstash_async
    - logstash_async.worker
```
which would have found the missing `limits`.

---

Ping here.

---

> Ping here.

Ughm, ping? @dopplershift I believe now it must be ready to merge.